src/simulation/context.tsx

```diff
@@ -25,6 +26,9 @@ interface SimulariumContextType {
     handleStartExperiment: () => void;
     section: Section;
     setPage: (value: number) => void;
+    module: Module;
+    setModule: (value: Module) => void;
+    moduleLength: number;
```

Comment:
Can `moduleLength` be a property of `module`?

Reply:
I fix this in another PR